apps/client/src/shared/apis/config/interceptor.ts

```diff
@@ -0,0 +1,32 @@
+import { HTTP_STATUS_CODE } from '@shared/constants/api';
+import { AxiosError, InternalAxiosRequestConfig } from 'axios';
+import { HTTPError } from './http-error';
+import { USER_ID_KEY } from '@shared/constants/user-constants';
+
+interface ErrorResponse {
+  success?: boolean;
+  message?: string;
+  code?: number;
+}
```

Comment:
```suggestion
interface ErrorResponse {
  message?: string;
  code?: number;
}
```
success는 제거해주세요! 현재 Error Response Body에는 message, code 두 개만 존재해요.

Reply:
맞네요 ! 반영하겠습니다.